README.md

```diff
@@ -20,22 +21,40 @@ See [**sample CV**](cv.pdf) and [@jskherman's CV](https://go.jskherman.com/cv):
 
 ## Usage
 
-### With [typst.app](https://typst.app)
+`cv.typ` is intended to be used by importing `cv.typ` file from a "content" file
+([`example.typ`](example.typ) as example). This content file is used to call the functions which
+apply document style & show CV components and load CV data from a YAML file
+([`example.yml`](example.yml) as example). Inside the content file you can modify several style
+variables and even override existing function implementations to your heart's content.
 
-1. Upload the [`cv.typ`](cv.typ), [`utils.typ`](utils.typ), and [`cv.typ.yml`](cv.typ.yml) files to your Typst project.
-2. In your project, Start editing `cv.typ` for the layout and `cv.typ.yml` for the data.
+### With [Typst CLI](https://github.com/typst/typst) (Recommended)
 
-### With [Typst CLI](https://github.com/typst/typst)
+The recommended usage with Typst CLI is by adding `cv.typ` as [git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules). This way upstream changes can be pulled easily.
```

Comment:
Perhaps, we could use clearer language here? Someone might confuse `cv.typ` (this GitHub repository) and `cv.typ` (the Typst file).

```suggestion
The recommended usage with Typst CLI is by adding this `cv.typ` repository as a [git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules). This way, upstream changes can be pulled easily.
```

Reply:
Updated.